cpp/src/centrality/betweenness_centrality.cu

```diff
@@ -37,8 +38,9 @@ void betweenness_centrality_impl(raft::handle_t const &handle,
                                  bool normalize,
                                  bool endpoints,
                                  WT const *weight,
-                                 VT const number_of_sources,
-                                 VT const *sources)
+                                 VT number_of_sources,
+                                 VT const *sources,
+                                 VT total_number_of_sources)
```

Comment:
total_number_of_sources : can't we retrieve this if everyone bcast its local sources? That would make the API nicer I think if we don't add too many batch-specific parameters

Reply:
Updated in 89c95c8, C++ no longer expects `total_number_of_sources`